saltproc/serpent_depcode.py

```diff
@@ -108,6 +110,20 @@ def create_runtime_matfile(self, file_lines):
             Serpent2 runtime input file with updated material file path.
 
         """
+        burnable_materials_path, absolute_path = self.get_burnable_materials_file(file_lines)
+
+        # Create data directory
+        Path.mkdir(Path(self.runtime_matfile).parents[0], exist_ok=True)
+
+        # Get material cards
+        flines = self.read_plaintext_file(absolute_path)
+        self.get_burnable_material_card_data(flines)
+
+         # Create file with path for SaltProc rewritable iterative material file
+        shutil.copy2(absolute_path, self.runtime_matfile)
+        return [line.replace(burnable_materials_path, self.runtime_matfile) for line in file_lines]
+
+    def get_burnable_materials_file(self, file_lines):
```

Comment:
If this is a _private_ function, I will accept not having a docstring. Since users should never interact with it. In this case, the code is documentation. However, if it _is a private function_ it should have a single underscore at the beginning of the method name.